proof/refine/AARCH64/Finalise_R.thy

```diff
@@ -2391,15 +2394,14 @@ lemma deleteASIDPool_invs[wp]:
     apply (strengthen invs_asid_update_strg')
   apply (wp mapM_wp' getObject_inv loadObject_default_inv
               | simp)+
-  sorry (* FIXME AARCH64 VMID
-  done *)
+  done
 
 lemma deleteASID_invs'[wp]:
   "deleteASID asid pd \<lbrace>invs'\<rbrace>"
-  sorry (* FIXME AARCH64
-  unfolding deleteASID_def by (wpsimp wp: getASID_wp) *)
+  unfolding deleteASID_def
+  by (wpsimp wp: getASID_wp hoare_drop_imps simp: getPoolPtr_def)
 
-(* FIXME AARCH64 move next to valid_objs_valid_tcb *)
+(* FIXME AARCH64 move next to valid_objs_valid_tcb' *)
 lemma valid_objs_valid_tcb':
   "\<lbrakk> valid_objs' s ; ko_at' (t :: tcb) p s \<rbrakk> \<Longrightarrow> valid_tcb' t s"
```

Comment:
I think that this `FIXME` can be removed, or at least have the destination for the move changed to a specific location. It looks like the original one was based on `ARM_HYP`, where `valid_objs_valid_tcb'` is next to `valid_objs_valid_tcb`, while `AARCH64` currently doesn't have a copy of `valid_objs_valid_tcb`.